src/networking/Request.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare( strict_types = 1 );
+
+class Request {
+
+	/** @var string */
+	public $url;
+
+	/** @var int */
+	public $port;
+
+	/** @var array */
+	public $headers;
+
+	/** @var string */
+	public $body;
+
+	/** @var bool */
+	public $debug;
+
+	/** @var bool */
+	public $ssl_verification_enabled;
+
+	public function __construct( string $url, int $port, array $headers, string $body, bool $debug, bool $ssl_verification_enabled ) {
```

Comment:
I originally built this type because CodeClimate complained about the signature of the `enqueueRequest` having 6 arguments. I thought maybe it would not have in a constructor, but that's not the case.

I think having a type in between the enqueue method and the parameters is cool but I feel like if I were to extract another object here to have less parameters, it would be just to make Code Climate happy, not because we really need it.